crates/subspace-farmer/src/bin/subspace-farmer/main.rs

```diff
@@ -77,7 +77,7 @@ struct DsnArgs {
     #[arg(long, default_value = "/ip4/0.0.0.0/tcp/30533")]
     listen_on: Vec<Multiaddr>,
     /// Piece cache size in pieces.
-    #[arg(long, default_value = "65536")]
+    #[arg(long, default_value = "1000")]
```

Comment:
Fine for now, but this should rather be something like 1% of plotted space (and spread across multiple plots). Please create an issue for now if we don't have it yet, I'll elaborate on the design I imagine there.